README.md

```diff
@@ -14,15 +14,28 @@ Set up configuration in one of the following paths:
 
 A valid configuration should specify an environment, its system(s) and the
 system(s) details.
-Use the configuration below as placeholder.
-
+Use the configuration below as a minimal configuration file.
 ```
 environments:
     env_1:
         jenkins_system:
             system_type: ""
 ```
 
+For query purposes, add sources in the following format:
+
+```
+environments:
+  osp_phases:
```

Comment:
osp_phases is too specific for OpenStack. Perhaps "production_ci" or "production"? same for other keys